RUN-ON-DOCKER.md

```diff
@@ -0,0 +1,46 @@
+### Run FiloDB locally with kafka and cassandra on docker
+
+#### Start kafka, zookeeper and cassandra containers
+```
+docker-compose up
+```
+
+#### Create kafka topics 
+```
+docker exec -it broker kafka-topics --create --zookeeper zookeeper:2181 --replication-factor 1 --partitions 4 --topic timeseries-dev
+```
+```
+docker exec -it broker kafka-topics --create --zookeeper zookeeper:2181 --replication-factor 1 --partitions 4 --topic timeseries-dev-ds-1m
```

Comment:
Downsampling via kafka is not used anymore and deprecated. This topic is not needed anymore.